x/emissions/types/params.go

```diff
@@ -179,3 +221,27 @@ func validateTssEmissionPercentage(i interface{}) error {
 	}
 	return nil
 }
+
+func validateObserverSlashAmount(i interface{}) error {
```

Comment:
I think we can use the true type in the argument. We used to have interface{} since generic data is retrieved from the param store but we now have type safety